### ISSUE TITLE
Start short tutorial for MS Open Community Camp

### ISSUE THREAD

**Issue by hassanabidpk** [djangogirlscodecamp/msopencamptutorial#1]
After completing it, I will ask @jinpark  to review it.

**Comment by jinpark** [comment]
My suggestion is to use commits as milestones(sections) so you can jump back and forth without too much work.

**Comment by hassanabidpk** [comment]
Yeah right. :+1:   I was thinking to make it more interesting (DiningKode or something fun).. or Just doing Blog would be okay ?

**Comment by jinpark** [comment]
up to you :) both would be fine imo
